shared/common-adapters/avatar-line.tsx

```diff
@@ -24,7 +24,7 @@ const AvatarLine = (props: Props) => {
   const usernamesToShow = props.usernames.slice(0, props.maxShown)
   const extra = props.usernames.length - usernamesToShow.length
   const reverse = {horizontal: 'horizontalReverse', vertical: 'verticalReverse'} as const
-  const styles = styleMap[props.size][props.layout]
+  const styles = styleMap.get(props.size)![props.layout] ?? {}
```

Comment:
What is this exclamation mark?

Reply:
https://www.typescriptlang.org/docs/handbook/advanced-types.html#type-guards-and-type-assertions